pulpcore/cli/common/context.py

```diff
@@ -1,4 +1,4 @@
-from typing import Any, Dict, List, Tuple
+from typing import Any, ClassVar, Dict, List, Tuple, Type
```

Comment:
Nice. Did not know about these.

Reply:
I am learning a lot reading x9c4's PRs as well